Tasks/XamarinAndroidV1/package.json

```diff
@@ -21,7 +21,7 @@
     "@types/mocha": "^5.2.7",
     "@types/uuid": "^8.3.0",
     "@types/q": "^1.5.1",
-    "azure-pipelines-task-lib": "^3.0.6-preview.0",
+    "azure-pipelines-task-lib": "^3.1.2",
```

Comment:
Could you please if there are other task-lib changes between 3.0.6-preview.0->3.1.2 we need to check for these tasks for backward compatibility?